app-creativity/tenacity/autobuild/beyond

```diff
@@ -0,0 +1,5 @@
+abinfo "Removing helper for non-system prefixes (which do not conform to FHS) ..."
+rm -v "$PKGDIR"/usr/tenacity
+
+abinfo "Removing a conflicting icon with Audacity ..."
+rm -v "$PKGDIR"/usr/share/pixmaps/gnome-mime-application-x-audacity-project.xpm
```

Comment:
Please consider making this an alternative, lest the file manager would not be able to show the correct icon for associated MIME types.